packages/core/src/generator/index.ts

```diff
@@ -659,6 +660,7 @@ class UnoGeneratorInternal<Theme extends object = object> {
 
             // Extract variants from special symbols
             let variants = variantHandlers
+            let entryMeta = { ...meta }
```

Comment:
```suggestion
            let entryMeta = meta
```

I think we can avoid copy when not necessary